2.22-RC release anouncement with illustrations (#134)
geoserver/geoserver.github.io: _posts/2022-10-19-geoserver-2-22-RC-released.md

This is a release candidate intended for public review and feedback.

Thanks to Jody Garnett (GeoCat) for making this release candidate.

### Release candidate public testing and feedback

Testing and providing feedback on releases is part of the open-source social contract. The development team (and their employers and customers) are responsible for sharing this great technology with you.

*The collaborative part of open-source happens now - we ask you to test this release candidate in your environment and with your data. Try out the new features, double check if the documentation makes sense, and most importantly let us know!**

*If you spot something that is incorrect or not working do not assume it is obvious and we will notice. We request and depend on your [email](https://geoserver.org/comm/) and [bug reports](https://geoserver.org/issues/) at this time. If you are working with [commercial support](https://geoserver.org/support/) your provider is expected to participate on your behalf.*

> Keeping GeoServer sustainable as a long term community commitment. If you are unable to contribute time, [sponsorship options](https://github.com/geoserver/geoserver/wiki/Sponsor) are available via OSGeo.

Reviewer: The first sentence doesn't make sense to me.

Possibly
```suggestion
Keeping GeoServer sustainable requires a long term community commitment. If you are unable to contribute time, [sponsorship options](https://github.com/geoserver/geoserver/wiki/Sponsor) are available via OSGeo.
```
?

### Natural Earth GeoPackage and workspace

> The sample [data directory](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22-RC/geoserver-2.22-RC-data.zip/download) now includes a small geopackage generated from [Natural Earth](https://www.naturalearthdata.com) data. These layers are a good example with multiple styles, and complete descriptions from the natural earth project.

Reviewer: ```suggestion
The sample [data directory](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22-RC/geoserver-2.22-RC-data.zip/download) now includes a small geopackage generated from [Natural Earth](https://www.naturalearthdata.com) data. These layers are good examples with multiple styles, and include complete descriptions from the Natural Earth project.
```

![World map](/img/posts/2.22/world-map-rc.png) <br/>

> Thanks to Jody Garnett (GeoCat) and for the attendees of the FOSS4G [GeoServer Beginner Workshop](https://docs.google.com/presentation/d/1fbPLN-1Cs95WK-IxDG1PxCEKyHwFbNBGNkkomxmLr0Y/edit?usp=sharing).

Reviewer: ```suggestion
Thanks to Jody Garnett (GeoCat) and to the attendees of the FOSS4G [GeoServer Beginner Workshop](https://docs.google.com/presentation/d/1fbPLN-1Cs95WK-IxDG1PxCEKyHwFbNBGNkkomxmLr0Y/edit?usp=sharing).
```

### User Manual Getting Started updated

> The user manual has been revised with:

Reviewer: ```suggestion
The user manual has been revised. Changes include:
```

> * [Getting started](https://docs.geoserver.org/latest/en/user/gettingstarted/index.html) revised with new sections for geopackage, image, layer group and style.

Reviewer: ```suggestion
* [Getting started](https://docs.geoserver.org/latest/en/user/gettingstarted/index.html) has been updated and includes with new sections for GeoPackage, image, layer group and style.
```

  
* [Tutorials](https://docs.geoserver.org/latest/en/user/tutorials/index.html) now provides an index of all tutorials across the user manual 

Thanks to Jody Garnett (GeoCat) and the attendees of the FOSS4G [GeoServer Beginner Workshop](https://docs.google.com/presentation/d/1fbPLN-1Cs95WK-IxDG1PxCEKyHwFbNBGNkkomxmLr0Y/edit?usp=sharing).

### Welcome Page Updates

**Layout**

The welcome page description provides a summary of the workspaces and layers available to the current user.

> The header includes welcome message and a link to the organization providing the service.

Reviewer: ```suggestion
The header includes a welcome message and a link to the organization providing the service.
```

> Each web services is listed using the service title as a heading, followed by the service abstract as a description. The protocols provided by the service are displayed as blocks linking to web service URL. These are the URLs used to access the service in a desktop or web application.

Reviewer: ```suggestion
Each web service is listed using the service title as a heading, followed by the service abstract as a description. The protocols provided by the service are displayed as blocks linking to the web service URL. These are the URLs used to access the service in a desktop or web application.
```

### Style format

The styles list provides a **Format** column indicating the format used.

![World map](/img/posts/2.22/styles.png) <br/>

Thanks to Mohammad Mohiuddin Ahmed for this change.

### CSW ISO and Metadata extension

To support the use of the CSW module the [Metadata extension](https://docs.geoserver.org/latest/en/user/extensions/metadata/index.html) provides a tab for editing metadata as part of layer configuration. It also provides a REST API for bulk metadata activities including importing from GeoNetwork.

The [CSW ISO Metadata](https://docs.geoserver.org/latest/en/user/extensions/csw-iso/index.html) profile is now available as an extension.  

![Metadata tab](/img/posts/2.22/metadata.png) <br/>

For background information see [GSIP-211](https://github.com/geoserver/geoserver/wiki/GSIP-211).

Thanks to Niels for for this work.

### Significant improvements in raster rendering performance

Raster rendering performance has increased significantly for two specific use cases:

* GeoTIFF hyperspectral images, with hundreds of bands, and band interleaved structure 
* Mosaicking hundreds of small images

> [Hyperspectral sensors](https://en.wikipedia.org/wiki/Hyperspectral_imaging) collect information at a very high spectral resolution, producing images with hundreds of bands. The typical pixel interleaved layout, where all the bands of a single pixel are stored toghether, is particuarly inefficient while rendering a false color image,

Reviewer: ```suggestion
[Hyperspectral sensors](https://en.wikipedia.org/wiki/Hyperspectral_imaging) collect information at a very high spectral resolution, producing images with hundreds of bands. The typical pixel interleaved layout, where all the bands of a single pixel are stored together, is particularly inefficient while rendering a false color image,
```

> where only three of them are used. A band interleaved, where each band is stored in a separate bank, is more efficient. GeoServer used to load

Reviewer: ```suggestion
where only three of them are used. A band interleaved, where each band is stored in a separate bank, is more efficient. GeoServer previously loaded
```

used to be slow and very memory intensive (going with the square of the output image size). 
The implementation has been improved so that the memory used in now linear with the output image size, and the amount of processing
has been reduced as well, providing again a couple of orders or magnitude speed up when mosaicking several hundreds small images.

![Mosaicking many little images](/img/posts/2.22/African_mosaic_ESA360518.jpg) <br/>

### Community modules news

News about community modules improvements, and new community modules you'll find in the 2.22.x series.

#### COG reader support for Azure

The COG reader community module now supports COGs stored in Azure as well.
The location of the COG can be provided as a HTTP(s) link, while eventual access credentials should be provided as system properties:

System Property | Description
-------------- | -----------
azure.reader.accountName | The Azure account name
azure.reader.accountKey | The Azure account key
azure.reader.container | The Azure container for the blobs
azure.reader.prefix | The optional prefix folder for the blobs

#### STAC datastore and mosaicking

> A new community module, [STAC datastore](https://docs.geoserver.org/latest/en/user/community/stac-datastore/index.html, allows connecting

Reviewer: ```suggestion
A new community module, [STAC datastore](https://docs.geoserver.org/latest/en/user/community/stac-datastore/index.html, supports connecting
```
